src/components/listItems/listItem.scss

```diff
@@ -0,0 +1,36 @@
+.list-item{
```

Comment:
Desing is not matching figma desing 

- Fonts are not equal 
- Line between items not require 
- Size of spacing not the same 
-  color of the fonts 
- not suppose to be a Card is only the list a card component is already created 
- extra border on last item remove it 
- item should be exact as figma desing 
Expected 
<img width="909" alt="Screen Shot 2022-09-26 at 9 56 12" src="https://user-images.githubusercontent.com/6474358/192310303-a62b8804-e949-46d2-957f-226b01de952a.png">
actual 

![image](https://user-images.githubusercontent.com/6474358/192310634-f2e1aa78-0918-4765-b4f7-596663f58637.png)